engine/runtime/src/main/scala/org/enso/interpreter/instrument/job/UpsertVisualisationJob.scala

```diff
@@ -118,39 +111,113 @@ class UpsertVisualisationJob(
     )
   }
 
-  private def replyWithModuleNotFoundError()(implicit
+  private def replyWithModuleNotFoundError(module: String)(implicit
     ctx: RuntimeContext
   ): Unit = {
     ctx.endpoint.sendToClient(
-      Api.Response(
-        requestId,
-        Api.ModuleNotFound(config.visualisationModule)
-      )
+      Api.Response(requestId, Api.ModuleNotFound(module))
     )
   }
 
+}
+
+object UpsertVisualisationJob {
```

Comment:
Upsert!? Strange word...

Reply:
upsert = update + insert, pretty common in db world